src/librustc_codegen_ssa/back/linker.rs

```diff
@@ -481,10 +481,12 @@ impl<'a> Linker for GccLinker<'a> {
         match strip {
             Strip::None => {}
             Strip::Debuginfo => {
-                self.linker_arg("--strip-debug");
+                // MacOS linker does not support longhand argument --strip-debug
```

Comment:
Can we use `cfg` guard for macOS case?

Reply:
Yes, but why, if the shorthand versions work in both macOS and non-macOS cases.